docs/index.md

```diff
@@ -5,60 +5,49 @@ page_keywords: documentation, docs,  docker, compose, orchestration, containers
 
 ## Overview
 
-Compose is a tool that allows you to orchestrate multiple Docker containers.
-With Compose, you can build clusters of containers which provide the resources
-(services, volumes, etc.) needed to build and run a complete distributed
-application.
+Compose is a tool for defining and running complex applications with Docker. It
+lets you define a multi-container application in a single file, then spin it up
+in a single command which does everything that needs to be done to get it
+running.
 
-You can use Compose to build your app with containers hosted locally, or on a
-remote server, including cloud-based instances - anywhere a Docker daemon can
-run. Its primary use case is for development environments, but it can be used
-just as easily for staging or CI.
+Compose is especially good for development environments, staging servers and CI.
+It can be used in production, but it isn't (yet) recommended that you do so.
+We're actively working on making it better in that regard - see the
+[roadmap](https://github.com/docker/fig/blob/master/ROADMAP.md).
 
-Using Compose is basically a three-step process.
-
-First, you define your app's environment with a `Dockerfile` so it can be
-reproduced anywhere:
-
-    FROM python:2.7
-    WORKDIR /code
-    ADD requirements.txt /code/
-    RUN pip install -r requirements.txt
-    ADD . /code
-
-Next, you define the services that make up your app in `docker-compose.yml` so
-they can be run together in an isolated environment:
+To use Compose, you define the services that make up your app in
+`docker-compose.yml`, using standard Docker configuration options:
 
 ```yaml
 web:
   build: .
   command: python app.py
-  links:
-   - db
   ports:
-   - "8000:8000"
-db:
-  image: postgres
+   - "5000:5000"
+  volumes:
+   - .:/code
+  links:
+   - redis
+redis:
+  image: redis
 ```
 
-(No more installing Postgres on your laptop!)
-
-Lastly, run `docker-compose up` and Compose will start and run your entire app.
+Then run `docker-compose up` and Compose will start and run your entire app,
+building or pulling whatever images it needs.
 
-Compose includes commands to:
+Compose has commands for managing the whole lifecycle of your application:
 
  * Start, stop and rebuild services
  * View the status of running services
- * tail the log output of running services
- * run a one-off command on a service
+ * Stream the log output of running services
+ * Run a one-off command on a service
 
 
 ## Quick start
 
-Let's get started with a walkthrough of getting a simple Python web app running
-on Compose. It assumes a little knowledge of Python, but the concepts
-demonstrated here should be understandable even if you're not familiar with
-Python.
+Let's get a simple Python web app running on Compose. It assumes a little
```

Comment:
Why are we getting a simply Python web app running now? Necessity? Whim? Curiosity? The reader needs context (e.g., "Let's get started" or "Let's see how this works by...", etc.) to know why we are doing what we are doing, when.